tests/python/pants_test/binaries/test_binary_util.py

```diff
@@ -165,7 +165,7 @@ def uname_func():
 
     with self.assertRaisesRegexp(BinaryUtil.MissingMachineInfo,
                                  r'Pants has no binaries for vms'):
-      binary_util._select_binary_base_path("supportdir", "name", "version", uname_func=uname_func)
+      binary_util._select_binary_base_path("supportdir", "version", "name", uname_func=uname_func)
```

Comment:
should all of the other callsites for `_select_binary_base_path` in this file invert `name`<->`version` too?

that seems to be the method signature:

```
def _select_binary_base_path(self, supportdir, version, name, uname_func=None):
```

so it's puzzling why other tests in this file pass them in the wrong order.

Reply:
Yes. Done.